roles/metering/defaults/main.yml

```diff
@@ -0,0 +1,35 @@
+---
+##########################
+# docker
+
+docker_network_mtu: 1500
+
+##########################
+# operator user
+
+operator_user: dragon
+operator_group: "{{ operator_user }}"
+
+##########################
+# metering
+
+metering_install_type: container
+metering_version: xena
+
+##########################
+# container
+
+metering_configuration_directory: /opt/metering/configuration
+metering_data_directory: /opt/metering/data
+metering_docker_compose_directory: /opt/metering
+
+metering_tag: "{{ metering_version }}"
```

Comment:
Why not simply metering_tag: zed here as default? This way we do not have to introduce the metering_version parameter.

Reply:
Thanks. I set the metering_tag to zed